src/homework7/StringAndSymbols.java

```diff
@@ -0,0 +1,87 @@
+package homework7;
+
+import org.w3c.dom.ls.LSOutput;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.OptionalInt;
+import java.util.Scanner;
+
+public class StringAndSymbols {
+
+//2. Створити метод findSymbolOccurance. Метод приймає як параметр рядок і символ.
+//Необхідно обчислити, скільки разів символ зустрічається в переданому рядку і повернути це числове значення.
+    public static void findSymbolOccurance(String string, char chr) {
+        int count = 0;
+        for (var i = 0; i<string.length(); i++) {
+            if (string.charAt(i) == chr) {
+                count++;
+            }
+        }
+        System.out.println("result of findSymbolOccurance() with string \""+string+"\" and char \'"+chr+"\' is "+count);
+    }
+//3. Створити метод findWordPosition. Метод приймає як параметри два рядки (source, target).
+//Необхідно з'ясувати, чи є target (підрядок) частиною рядка source. Якщо так, тоді повернути номер позиції (індекс)
+//першого елемента підрядка у рядку, інакше повернути -1.
+    public static void findWordPosition(String source, String target) {
+        System.out.println("result of findWordPosition() with source \""+source+"\" and target \""+target+"\" is position "+source.indexOf(target));
+    }
+//4. Створити метод stringReverse. Метод приймає як параметр - рядок. Необхідно її розгорнути та повернути змінений варіант.
+    public static String stringReverse(String string) {
+        String result = new String();
+        for (var i = string.length()-1; i>=0; i--) {
+            result += string.charAt(i);
```

Comment:
2. "string.length()-1" - new objects created 
`result += string.charAt(i);` - each concatenation will create a new string

Reply:
Use StringBuilder or Stringkoiner in case of multiple string concatenate/modification